schemas/spacefile/spacefile.v0.schema.json

```diff
@@ -118,25 +114,42 @@
           "description": "Routes that will be available publicly",
           "type": "array",
           "minItems": 1,
+          "uniqueItems": true,
           "items": {
             "type": "string"
           }
         },
+        "public": {
+          "description": "If the Micro should be available to the public",
+          "type": "boolean",
+          "default": false
+        },
         "actions": {
           "description": "Tasks that run on triggers like a schedule",
           "type": "array",
           "minItems": 1,
+          "uniqueItems": true,
```

Comment:
I don't think that this work for object type.

Reply:
Right, it doesn't.